key.core/src/main/java/de/uka/ilkd/key/java/Recoder2KeY.java

```diff
@@ -1254,8 +1254,12 @@ public static void reportError(String message, Throwable t) {
             throw (PosConvertException) cause;
         }
 
-        var pos = extractPositionInfo(cause.toString());
-        reportErrorWithPositionInFile(message, cause, pos.second, pos.first);
+        Pair<String, Position> pos = extractPositionInfo(cause.toString());
```

Comment:
Here, it would be useful to avoid `Pair` and use one of the location classes available in KeY.

Reply:
I'd like to delegate this to @jwiesler who changed this from `Position` to `Pair`.
This is merely a bugfix to avoid the NPE which would arise on almost all input with a syntax error.